providers/softlayer/src/main/java/org/jclouds/softlayer/compute/strategy/SoftLayerComputeServiceAdapter.java

```diff
@@ -272,7 +272,7 @@ public int compare(Hardware h1, Hardware h2) {
                     .compare(getBootableDeviceType(h1), getBootableDeviceType(h2));
             if (!volumes1.isEmpty() && !volumes2.isEmpty() && volumes1.size() == volumes2.size()) {
```

Comment:
@andrewgaul thanks for the comment but I'm not sure I got your suggestion. Could you rephrase it, please?